CMakeLists.txt

```diff
@@ -111,6 +114,11 @@ find_package(Doxygen 1.8.5)
 
 find_package(GTest 1.6.0)
 
+if (USE_EXTERNAL_MFEM)
```

Comment:
I think I only understand this partly. Do you mean that we can streamline the input `-DMFEM_DIR` and `-DUSE_EXTERNAL_MFEM`? This is what I have streamlined in a new commit. If `-DMFEM_DIR` is not empty it will set the boolean `USE_EXTERNAL_MFEM` to `true`.

Reply:
This is removed.